src/Data/Functor/Rep.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE CPP #-}
+{-# LANGUAGE PolyKinds #-}
```

Comment:
Guard this with `#if __GLASGOW_HASKELL__ >= 706 ... #endif`, since `PolyKinds` didn't work reliably on versions of GHC older than 7.6.